tests/test_psycopg2.py

```diff
@@ -16,6 +17,10 @@ class UserBlogSummary(NamedTuple):
 RECORD_CLASSES = {"UserBlogSummary": UserBlogSummary}
 
 
+def test_version():
+    assert re.match(r"^2\.", psycopg2.__version__)
```

Comment:
Hi Fabien, I've started looking at aiosql to use it in a project and while looking around the repo I saw your PR and was interested because I'm also using psycopg3.

While reading the PR I saw the code above and wondered if you might think this alternative is simpler than using a regular expression:

```python
assert psycopg2.__version__.startswith('2.')
```

Thanks for working on this PR!

Reply:
Yes, I could have avoided a re.

Reply:
I have done this change.